### ISSUE TITLE
Problema IRsend does not name a type

### ISSUE THREAD

**Issue by babiribeiro** [witnessmenow/ESP8266-IR-Remote-Example#4]
Olá pessoal, este é meu primeiro post, então se faltar alguma informação me perdooem :)

Eu estou usando um ESP8266 para clonar o controle de um ar-condicionado e ajustar a temperatura do mesmo caso ele saia de uma temperatura pré-estabelecida.

Quando eu tento compilar o código ele aparece o seguinte erro:

Arduino: 1.8.8 (Windows 10), Placa:"Generic ESP8266 Module, 80 MHz, Flash, Enabled, ck, 26 MHz, 40MHz, QIO, 512K (no SPIFFS), 2, v2 Lower Memory, Disabled, None, Only Sketch, 115200"

Opções de compilação alteradas, recompilando tudo
ESP8266_Gmail_Sender_quase_final:5:1: error: 'IRsend' does not name a type

 IRsend irsend(14); //pino D5

 ^

C:\Users\Barbara\Documents\Arduino\libraries\ESP8266_Gmail_Sender_quase_final\ESP8266_Gmail_Sender_quase_final.ino: In function 'void setup()':

ESP8266_Gmail_Sender_quase_final:75:3: error: 'irsend' was not declared in this scope

   irsend.begin(); //INICIALIZA A FUNÃ‡ÃƒO

   ^

C:\Users\Barbara\Documents\Arduino\libraries\ESP8266_Gmail_Sender_quase_final\ESP8266_Gmail_Sender_quase_final.ino: In function 'bool ligar()':

ESP8266_Gmail_Sender_quase_final:171:3: error: 'irsend' was not declared in this scope

   irsend.sendRaw(ON, tamanho, frequencia);

   ^

C:\Users\Barbara\Documents\Arduino\libraries\ESP8266_Gmail_Sender_quase_final\ESP8266_Gmail_Sender_quase_final.ino: In function 'bool desligar()':

ESP8266_Gmail_Sender_quase_final:190:3: error: 'irsend' was not declared in this scope

   irsend.sendRaw(OFF, tamanho, frequencia);

   ^

exit status 1
'IRsend' does not name a type

Já pesquisei sobre esse problema e dizem para apagar a biblioteca RobotIRremote, que ela dá conflito com as biblioteca do Infravermelho, mas eu nem estou com ela baixada. 

Alguém consegue me ajudar?

Obrigada!

**Comment by ghost** [comment]
The problem is not in "RobotIRremote" library - it's caused by incompatibility of newer versions  of IRremoteESP8266 with older ones. There's two ways to fix it:
1) Download [latest 1.x version (1.2.0)](https://github.com/markszabo/IRremoteESP8266/releases/tag/v1.2.0) of IRremoteESP8266 and use it instead of installed one
2) Better - upgrade the code to 2.0 version by [easy tutorial](https://github.com/markszabo/IRremoteESP8266/wiki/Upgrading-to-v2.0), what I've done in [my fork](https://github.com/YChvanov/ESP8266-IR-Remote-Example). As my IR-LED died I can't say if this works or not, but at least it compiles. If it's okay I'll PR my changes.

tl;dr try my fork
